### PR TITLE
Display talk description and speaker info in tentative schedule.

### PR DESCRIPTION
**Pull request by priyankt68**
### How

1. Added a ```div``` with CSS class ```tg-description``` for loading description from ```js/main.js```'s ```updateScheduleForADay()```. This is set to ```display:none``` by default.

2. On clicking talk's title, ```tg-description```'s div is toggled using jQuery's ```slideToggle()``` method.

Note - Refactored ```each_row``` variable to remove redundancy alongwith formatting changes.

### CODE REVIEW

**Review by rajataaron — COMMENTED** [review]
Looks good to me

**Comment by CuriousLearner** [comment]
Description is not there in this PR as suggested in the title

File: js/main.js [hunk 1]
```diff
@@ -457,10 +462,15 @@ jQuery(document).on('ready', function() {
...
-                              <time datetime="2017-12-12">`+ nrow[0] +`</time>
-                              <div class="tg-title">
-                                <h2>`+ nrow[1] +`</h2>
-                              </div>
+                              <time datetime="2017-12-12">`+ nrow[0] + `</time>
+                              <div class="tg-title talk_id "  data-id='talk` + nrow[4] + `' >
+                              <h2>` + nrow[1] + `</h2>
+                            </div>
+                            
+                            <div class="tg-description" id='desc` + nrow[4] + `'>
```
Comment (priyankt68) [inline js/main.js:470]: @CuriousLearner The div with ```tg-description``` loads talk description from ```data/tracks.json.```. By default, it's hidden (mentioned in ```css/main.css```)

Comment (CuriousLearner) [reply js/main.js:470]: Yeah, I've seen that. What I meant was that `tracks.json` wasn't updated with the details.

File: css/main.css [hunk 2]
```diff
@@ -4974,3 +4974,4 @@ body.tg-comingsoonpage .tg-foorterbar .tg-socialicons{padding:0;}
   color: white;
 }
 
+.tg-description{ display: none}
```
Comment (CuriousLearner) [inline css/main.css]: Just leave a blank line at the end of the file.

Comment (priyankt68) [reply css/main.css]: Sure. Will do.

File: js/main.js (hunk 1 above)
Comment (priyankt68) [reply js/main.js:470]: Oh. Okay.

Since there's a separate issue for it - #132, therefore I didn't.

File: css/main.css (hunk 2 above)
Comment (CuriousLearner) [reply css/main.css]: Once you do that, just ping me, and I'll merge the PR.

**Review by realslimshanky — APPROVED** [review]
LGTM

File: js/main.js (hunk 1 above)
Comment (realslimshanky) [reply js/main.js:470]: We'll have a 'cfp' key also when #137 will be merged, could you add also show 'CFP Proposal' with the value of that key. It'll take the user to that particular talk's CFP proposal. Also is the 'cfp' is empty make sure to not add the entire div with 'CFP Proposal'.

**Review by realslimshanky — COMMENTED** [review]
Just one more enhancement to the PR. Rest is okay.

File: css/main.css (hunk 2 above)
Comment (priyankt68) [reply css/main.css]: Done.

**Comment by priyankt68** [comment]
@realslimshanky Sure. Will make "CFP Proposal" related changes.

**Review by CuriousLearner — APPROVED** [review]
We will do those changes later.

Looks good to go for now.

**Pull request merged**